ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh

```diff
@@ -61,6 +61,6 @@ then
 fi
 
 openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-DISABLED_MONITOR_TESTS="apiserver-availability,apiserver-new-disruption-invariant,disruption-summary-serializer,external-service-availability,incluster-disruption-serializer,image-registry-availability,ingress-availability,pod-network-avalibility,service-type-load-balancer-availability"
+DISABLED_MONITOR_TESTS="apiserver-new-disruption-invariant,disruption-summary-serializer,incluster-disruption-serializer,pod-network-avalibility"
```

Comment:
thanks for the pr!

@xueqzhan do you know why `apiserver-new-disruption-invariant` is disabled ?

@xueqzhan also, do  you know the difference between `apiserver-new-disruption-invariant` and `apiserver-availability` and `external-service-availability` ?